crates/compiler/mono/src/layout.rs

```diff
@@ -2191,6 +2192,7 @@ fn lambda_set_size(subs: &Subs, var: Variable) -> (usize, usize, usize) {
                     }
                     stack.push((ext.var(), depth_any + 1, depth_lset));
                 }
+                FlatType::EffectfulFunc => internal_error!(),
```

Comment:
minor: would be good to add a message describing the issue, even a couple words

Reply:
Actually, this case is fine and should be ignored